src/mpl/src/mem/mpl_trmem.c

```diff
@@ -15,6 +15,10 @@
 
 #include "mpl.h"
 
+#ifdef MPL_HAVE_ASSERT_H
+#include <assert.h>
```

Comment:
Hmm...that's weird. I don't know where that went. I guess I must have taken out the call to `assert` after I added the include. I'll pull this.